docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   hypernode:
     container_name: hypernode
-    image: docker.hypernode.com/byteinternet/hypernode-buster-docker-php${PHP_VERSION_SHORT:-81}-mysql57
+    image: docker.hypernode.com/byteinternet/hypernode-${IMAGE_OS:-buster}-docker-php${PHP_VERSION_SHORT:-81}-mysql57
```

Comment:
still using buster img?

Reply:
Yes it falls back to buster when PHP is not 8.4, I didn't want to make the PR bigger than it already is, so we can test the rollout of bookworm across CI/CD on a later stage. See https://github.com/ByteInternet/hypernode-deploy/pull/130/files#diff-e6214a8ed3463ef1ce3df7649af0c447f2c48c8e282370c9ef9a90df07a0abbcR7